README.adoc

```diff
@@ -3,21 +3,59 @@
 image:http://join.spinnaker.io/badge.svg[Slack Status,link=http://join.spinnaker.io]
 image:https://travis-ci.org/spinnaker/spinnaker.svg?branch=master["Build Status", link="https://travis-ci.org/spinnaker/spinnaker"]
 
-Spinnaker is a continuous delivery platform for releasing changes with high velocity and confidence.
+= Welcome to the Spinnaker Project
 
-This repository is used only to store and organize ancillary configuration, build, and testing tools for the Spinnaker project. The core code making up Spinnaker's microservices is found in the other https://github.com/spinnaker[Spinnaker repositories].
+Spinnaker is an open-source continuous delivery platform for releasing software changes with high velocity and confidence.
+Through a powerful abstraction layer, Spinnaker provides compelling tooling that empowers developers to own their application code from commit to delivery.
+
+Inside Spinnaker, using a GUI (graphical user interface), or as code, manage deployments and cloud resources such as: 
```

Comment:
"Manage deployments in Spinnaker using the graphical user interface (GUI) or with config-as-code. A deployment may be one or all of these:"